clients/admin-ui/src/features/system/SystemFormTabs.tsx

```diff
@@ -142,6 +146,8 @@ const SystemFormTabs = ({
      */
     if (isCreate) {
       dispatch(setActiveSystem(undefined));
+      dispatch(setSuggestions("hiding"));
```

Comment:
You'll want to make sure that this works with Allison's recent changes: 

https://github.com/ethyca/fides/pull/4347